### PR TITLE
New default layout

### PR DESCRIPTION
**Pull request by brandelune**
This PR keeps the layout related commits from https://github.com/omegat-org/omegat/pull/386

### CODE REVIEW

**Comment by Kazephil** [comment]
This one looks good here!

**Comment by brandelune** [comment]
@Kazephil I just added some code to check the available screenspace so that OmegaT stays within the empty available space and does not hide behind toolbars or whatnot.

@miurahr please, check the validity of the code. It works, but it might not be the best way to put it.

**Comment by Kazephil** [comment]
> I just added some code to check the available screenspace so that OmegaT stays within the empty available space and does not hide behind toolbars or whatnot.

Nice!

**Comment by brandelune** [comment]
It works on macOS with the Dock on the left/right at the bottom. There is only a small glitch when the Dock is on the left, but I don't want to bother about that. The code is generic enough. Could you test that on Windows ? You'd need to remove your default OmegaT preference folder first.

File: src/org/omegat/gui/main/MainWindowUI.java [hunk 1]
```diff
@@ -267,7 +269,34 @@ public void mouseClicked(MouseEvent e) {
...
+		
+		if (omegatHeight > screenHeight) {
+			omegatHeight = screenHeight;
+		}
+		
+		// Attempt to center the OmegaT main window on the screen
+		int omegatLeftPosition = (screenWidth - omegatWidth)/2;
+
+		Rectangle defaultWindowSize = new Rectangle(omegatLeftPosition, 0, omegatWidth, omegatHeight);
+
```
Comment (miurahr) [inline src/org/omegat/gui/main/MainWindowUI.java:298]: Please format indent.  It may be good idea to run spotlessChangedApply task

**Review by miurahr — CHANGES_REQUESTED** [review]

**Comment by miurahr** [comment]
Now rebased and merged into master. Thank you for hard work!

**Pull request closed without merging**